Airwallex/Payment/Sources/CardPaymentConsentSectionController.swift

```diff
@@ -254,6 +276,33 @@ class CardPaymentConsentSectionController: SectionController {
     func updateItemsIfNecessary() {
         consents = methodProvider.consents.filter { $0.paymentMethod != nil }
     }
+    
+    func sectionWillDisplay() {
+        AnalyticEvent.log(
+            paymentMethodView: .card,
+            extraInfo: [
+                .subtype: Self.subType
+            ]
+        )
+        
+        cvcInputToken = NotificationCenter.default.publisher(for: UITextField.textDidBeginEditingNotification)
+            .filter { [weak self] in
+                guard let self,
+                      self.mode == .payment,
+                      let object = $0.object as? UITextField,
+                      let cell = context.cellForItem(Items.cvcField) as? InfoCollectorCell else {
+                    return false
+                }
+                return object.isDescendant(of: cell)
```

Comment:
Shall we observe a published object or passthrough object directly from `InfoCollectorCell` instead? This checking might be less efficient and convoluted.